config/puma.rb

```diff
@@ -12,15 +12,15 @@
 # Starting with a low number of workers and threads provides adequate
 # performance for most applications, even under load, while maintaining a low
 # risk of overusing memory.
-workers Integer(ENV.fetch('WEB_CONCURRENCY', 2))
-threads_count = Integer(ENV.fetch('MAX_THREADS', 2))
+workers Integer(Figaro.env.web_concurrency || 2)
```

Comment:
hmmm interesting question. Looks like this was added back when this project was first created / deployed with Cloud.gov. Perhaps I will leave as `ENV` and follow up with a PR removing refs to Puma?